backend/api/Areas/Admin/Mapping/AccessRequest/UserMap.cs

```diff
@@ -9,21 +9,21 @@ public class UserMap : IRegister
     {
         public void Register(TypeAdapterConfig config)
         {
-            config.NewConfig<Entity.User, Model.UserModel>()
+            config.NewConfig<Entity.PimsUser, Model.UserModel>()
                 .Map(dest => dest.Id, src => src.Id)
-                .Map(dest => dest.KeycloakUserId, src => src.KeycloakUserId)
-                .Map(dest => dest.BusinessIdentifier, src => src.BusinessIdentifier)
+                .Map(dest => dest.KeycloakUserId, src => src.GuidIdentifierValue)
+                .Map(dest => dest.BusinessIdentifierValue, src => src.BusinessIdentifierValue)
```

Comment:
In general I tried to leave the contract with the frontend unchanged, but there is a leaky abstraction with api fields that are sortable in table headers. Those fields must have the same name as the field on the backend in order for the search to work.